### DISCUSSION TITLE
Connect to an external database

### DISCUSSION THREAD

**Discussion by omusil24** [supabase/supabase#813] [Questions] [answered]
Hi,

is it possible to somehow connect to an external database with Supabase or use my own one instead of the Supabase-hosted one? I didn't find any info about this in the documentation or in the Dashboard. And if it's not possible now, will it be possible in the future?

**Comment by silentworks** [comment, marked as answer]
@omusil24 this is not currently possible at the moment from the superbase interface, but there is a docker image that allows you to setup your own supabase instance wherever you please while using your own Postgres database instance.

This thread has some information on it https://github.com/supabase/supabase/discussions/219
